src/lib/types/order.ts

```diff
@@ -0,0 +1,17 @@
+
+// Add new type for orders
+export interface EngineOrder {
+    orderId: string;
+    clientOrderId: string;
+    symbol: string;
+    side: 'BUY' | 'SELL';
```

Comment:
```suggestion
    orderSide: 'BUY' | 'SELL';
    positionSide: 'LONG' | 'SHORT' | 'NONE'
```

Should we add position side too? Since with hedge mode with perp futures trading, often orders need to specify the position side they're intended for. If so, worth renaming "side" too so there's no confusion with order vs position side.